restart_process/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+)
+
+var defaultWatchFile = "/.restart-proc"
+var watchFile = flag.String("watchfile", "", "File that entr will watch for changes; changes to this file trigger entr to rerun the command(s) passed")
+
+func main() {
+	flag.Parse()
+
+	args := os.Args[1:]
+	if *watchFile == "" {
+		*watchFile = defaultWatchFile
+	} else {
+		// user passed this arg, make sure we don't pass it along to entr
+		args = os.Args[2:]
+	}
+	fmt.Println("Hello, world.")
```

Comment:
👀

Reply:
(and the line below is probably also debug logging?)

Reply:
whooop.